lock/src/main/java/org/infinispan/lock/impl/lock/ClusteredLockImpl.java

```diff
@@ -108,8 +114,26 @@ public void handleLockResult(Boolean result, Throwable ex) {
 
       protected abstract void handle(Boolean result);
 
-      private String createRequestId() {
-         return Util.threadLocalRandomUUID().toString();
+      protected abstract void forceFailed();
+
+   }
+
+   public class UnlockRequestHolder extends RequestHolder<Void> {
+
+      public UnlockRequestHolder(Object requestor, CompletableFuture<Void> request) {
+         super(requestor, request);
+      }
+
+      @Override
+      protected void handle(Boolean result) {
+         // Unlock result
+
```

Comment:
Was there supposed to be something here?

Reply:
I'm wondering still if I need to hold unlocks too. I would like to just trace the unlock result